### ISSUE TITLE
Current semester GPA calculation + cumulative GPA calculation seems to be broken

### ISSUE THREAD

**Issue by Suhas-13** [gary-kim/saspes#221]
Both calculations are broken, likely due to some change to PowerSchool UI.

Will try and fix it soon.

**Comment by gary-kim** [comment]
Right. Also, I'll get the repo moved to an org and give you admin rights for it. Give me a bit
